src/routes.js

```diff
@@ -1,10 +1,10 @@
-import Overview from "./views/Overview.vue";
-import Dashboard from "./views/Dashboard.vue";
-const Organizations = () => import("./views/Organizations.vue");
-const Organization = () => import("./views/Organization.vue");
-const AddOrganization = () => import("./views/AddOrganization.vue");
-const EditOrganization = () => import("./views/EditOrganization.vue");
-const NotFound = () => import("./views/NotFound.vue");
+import Dashboard from "@/views/Dashboard.vue";
+const Overview = () => import("@/views/Overview.vue");
```

Comment:
Since we will load Overview by default I would not lazy load it.
In fairness the one view where it makes sense to lazy load is the Installation detail, since it's loading a lot of javascript due to the graph and in turn required libraries.

Reply:
ok